Source/UrlRequest_Windows.cpp

```diff
@@ -172,37 +168,57 @@ namespace UrlLib
 
         gsl::span<const std::byte> ResponseBuffer() const
         {
-            if (!m_responseBuffer)
+            if (m_fileResponseBuffer.size() > 0)
```

Comment:
Sorry, should have mentioned this in the previous comment.
```suggestion
            if (!m_fileResponseBuffer.empty())
```